### ISSUE TITLE
Mark each issue as resolved for myself, meaning I changed my password

### ISSUE THREAD

**Issue by pdehaan** [mozilla/blurts-server#1174]
Via Slack:

> **[Asa Dotzler 1:13 PM]:** https://mobile.twitter.com/webfairyjo/status/1160182494173519872
>
>> Joanie Waelti @webfairyjo
>> Hello @firefox , I have a suggestion for #firefoxmonitor, I'd like to be able to mark each issue as resolved for myself, meaning I changed my password. Could you please add this? Thanks :pray::slightly_smiling_face:
>
> I too would like such a feature. Monitor shows me things I've already fixed and I'd like to "resolve" those.

**Comment by changecourse** [comment]
Certainly worth discussing in more depth, but we are planning for this type of feature within the context of Lockwise, and handling resolving these breached accounts by automatically dismissing breach alerts associated with Saved Logins within the browser (because we can not only confirm, but automatically dismiss alerts as logins are changed)... that being said, not everyone is storing their logins within Firefox, so their may still be an opportunity here... 

It is valuable to maintain a historical record of breaches for each email, so I wouldn't want these to be dismissed, but perhaps there's a nice way to handle two states for the cards based on whether a user has 'resolved' passwords for various breached accounts.

**Comment by aelisya** [comment]
At least do 0 at "Password exposed across all breaches." for the password we have marked as resolved but let "Known data breach has exposed your information." to 1 by default.

**Comment by VasVadum** [comment]
I would rather not have to use Lockwise to mark a breach as solved. @changecourse 

A simple feature of being able to login to firefox monitor could be that it saves small bits of data on breaches that the user marks as resolved. 0 for unresolved, 1 for resolved. A binary storage value for each breach, would barely take any space at all for Mozilla I bet.

**Comment by joshpc** [comment]
To highlight how great this would be... as a user my expectation is that if I get an email with a breach, that it's something that I must immediately take care of. Today, I received a summary email detailing four breaches that I've already taken care of -- but had forgotten about. When I went through all of the breaches, I realized that I had just wasted a bunch of time cycling passwords.

It'd be great if I could mark a breach as "taken care of" and never receive _any_ communication about it ever again.
